db/migrate/20200326164755_change_oidc_audience_setting_type.rb

```diff
@@ -0,0 +1,27 @@
+class ChangeOidcAudienceSettingType < ActiveRecord::Migration[5.2]
+  def up
+    setting = Setting.find_by :name => 'oidc_audience'
+    return unless setting
+    if setting.value
+      setting.value = setting.value.split
+    else
+      setting.value = []
+    end
```

Comment:
There are two possible cases that we need to take into account here:
1. The setting has not been changed from the default value. In that case we shouldn't update the value at all but rather keep it as is - it will return the new default (`[]`) instead of the old default (`nil`). This can be identified by `setting[:value] == nil`. Depending on whether the default has been updated in the initialization, we might see `setting.value == nil` or `setting.value == []`, due to how the `Setting#value` method works. However since no action is required in this case we don't need to identify it - just handle the other case.
1. The user has defined a (string) value for the setting. In that case, we need to wrap the value in an array, i.e.: `setting.value = [setting.value] if setting.value.is_a?(String)`

Reply:
@tbrisker done.